shell/platform/windows/external_texture_gl.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_EXTERNAL_TEXTURE_GL_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_EXTERNAL_TEXTURE_GL_H_
+
+#include <stdint.h>
+
+#include <memory>
+
+#include "flutter/shell/platform/common/cpp/public/flutter_texture_registrar.h"
+#include "flutter/shell/platform/embedder/embedder.h"
+
+namespace flutter {
+
+typedef struct ExternalTextureGLState ExternalTextureGLState;
+
+// An adaptation class of flutter engine and external texture interface.
```

Comment:
This description isn't really clear to me, especially given that it doesn't implement an interface. Can you adjust this comment to more clearly describe this object's role in the texture system?